docker-compose.yml

```diff
@@ -118,38 +138,38 @@ services:
       - redis
       - rethink
     environment:
-      - ENV=development
-      - REDIS_URL=redis://redis:6379
-      - RETHINKDB_DB=place_development
-      - RETHINKDB_HOST=rethink
-      - SG_ENV=development
-      - TZ=$TZ
+      # Service Hosts
+      << : *redis-client-env
+      << : *rethinkdb-client-env
+      # Environment
+      << : *deployment-env
 
   # Support
 
-  init: # Idempotent initializer
-    image: placeos/init:beta
+  init:
+    image: placeos/init:${PLACE_INIT_TAG:-latest}
     container_name: init
     restart: on-failure
     networks:
       - placeos
     depends_on:
       - rubber-soul
     environment:
-      - ENV=development
-      - ES_HOST=elastic
-      - RETHINKDB_HOST=rethink
-      - RETHINKDB_DB=place_development
-      - PLACE_DOMAIN=localhost:8080
-      - PLACE_APPLICATION=backoffice
-      - PLACE_AUTH_HOST=auth:8080
-      - PLACE_USERNAME=Place Support (localhost:8080)
-      - PLACE_EMAIL=support@place.tech
-      - PLACE_PASSWORD=development
-      - TZ=$TZ
+      # User/Application Configuration
+      PLACE_DOMAIN: ${PLACE_DOMAIN:-localhost:8080}
+      PLACE_APPLICATION: ${PLACE_APPLICATION:-backoffice}
+      PLACE_AUTH_HOST: ${PLACE_AUTH_HOST:-auth:8080}
+      PLACE_USERNAME: ${PLACE_USERNAME:-Place Support (localhost:8080)}
+      PLACE_EMAIL: ${PLACE_EMAIL:-support@place.tech}
+      PLACE_PASSWORD: ${PLACE_PASSWORD:-development}
```

Comment:
There's [defaults for these in the init service](https://github.com/PlaceOS/init/blob/18e92ec4630d299eba20b3cb3adbe6537ba27a3a/src/start.cr#L5-L12) too. This means that we have three sets of references (here, .env and there) to maintain / chase for where config may originate from. Can this be reduced to one?